### PR TITLE
Add CMS HI release

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
How does this look, @tiborsimko? I just realised that I authored it as one sentence on one line. Will this break the display? I can switch to making a single line for each paragraph if needed.

### CODE REVIEW

**Comment by tiborsimko** [comment]
Thanks, I'll have a look and re-push if need be, together with any illustrating images (CC @katilp)

**Comment by katilp** [comment]
@tiborsimko I'm pushing an image, is it OK in #2972?

**Comment by tiborsimko** [comment]
@katilp Yes, no problem, it could be in a separate PR, I can then merge in proper order...

File: cernopendata/modules/fixtures/data/docs/cms-releases-heavy-ion-data/cms-releases-heavy-ion-data.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+CMS has already released 100% of the proton–proton data the collaboration recorded in 2010 and 2011.
+Therefore, with the release of the lead–lead data, all of the data CMS recorded in the first two years of LHC operation is in the public domain, in accordance with the collaboration open-data policy.
+
+Analysing data from the LHC requires specialised software and analysis environments.
+CMS is providing these in the form of Virtual Machine images as well as Docker container images, which come with all the necessary software.
+Further, in order for a new data explorer to familiarise themself with the kinds of analysis one can perform with the data, CMS has also provided usage examples.
+Those interested in specific advice can ask for it on the [official CERN Open Data forum](https://opendata-forum.cern.ch).
+
+“Preparing this release presented unique challenges to CMS,” notes Kati Lassila-Perini, who has lead the CMS Data Preservation and Open Access team since 2012.
```
Comment (katilp) [inline cernopendata/modules/fixtures/data/docs/cms-releases-heavy-ion-data/cms-releases-heavy-ion-data.md:22]: lead -> led ?

**Review by katilp — CHANGES_REQUESTED** [review]
One fix from Federica (lead -> led), another suggestion "all of the CMS open data are" -> "all of the CMS open data is" but we should keep "are"
Other comments may come.
@tiborsimko Will you add the figure /static/img/cms_hi_event_1_light.png (#2972)

File: cernopendata/modules/fixtures/data/docs/cms-releases-heavy-ion-data/cms-releases-heavy-ion-data.md (hunk 1 above)
Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/docs/cms-releases-heavy-ion-data/cms-releases-heavy-ion-data.md:22]: Yes, should be "led". Data are plural. :) Will update the PR.

Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/docs/cms-releases-heavy-ion-data/cms-releases-heavy-ion-data.md]: Done.

**Comment by tiborsimko** [comment]
I'm adding the figure now, so I can amend also the  "led" bit...

**Comment by tiborsimko** [comment]
FYI I have amended brief text for seach pages, pushed announcement date for Monday, added the leading image, amended feature numbers so that the News page will feature only three, wrapped long lines, repushing. Let's wait for other comments and plan for merge on Monday?

**Comment by RaoOfPhysics** [comment]
Hi @tiborsimko. I had already fixed the "led" bit in my latest commit. :)

Thanks for taking care of everything else! Excited for the release. :D

**Review by tiborsimko — APPROVED** [review]

**Pull request merged**